dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/category/CategoryOptionController.java

```diff
@@ -27,18 +27,45 @@
  */
 package org.hisp.dhis.webapi.controller.category;
 
+import java.util.Objects;
+import java.util.Set;
+
+import lombok.RequiredArgsConstructor;
+
+import org.hisp.dhis.association.IdentifiableObjectAssociations;
 import org.hisp.dhis.category.CategoryOption;
+import org.hisp.dhis.category.CategoryService;
 import org.hisp.dhis.schema.descriptors.CategoryOptionSchemaDescriptor;
 import org.hisp.dhis.webapi.controller.AbstractCrudController;
 import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
 
 /**
  * @author Morten Olav Hansen <mortenoh@gmail.com>
  */
 @Controller
 @RequestMapping( value = CategoryOptionSchemaDescriptor.API_ENDPOINT )
+@RequiredArgsConstructor
 public class CategoryOptionController extends AbstractCrudController<CategoryOption>
 {
+    private final CategoryService categoryService;
+
+    @ResponseBody
+    @GetMapping( value = "orgUnits" )
+    IdentifiableObjectAssociations getProgramOrgUnitsAssociations(
```

Comment:
Is the name of this method wrong, coming from copy-paste?

Reply:
oh yes, thank you for pointing it out :)